src/Core/src/Handlers/Entry/EntryHandler.Android.cs

```diff
@@ -274,5 +276,14 @@ internal void HideClearButton()
 			PlatformView.SetCompoundDrawablesWithIntrinsicBounds(null, null, null, null);
 			_clearButtonVisible = false;
 		}
+
+		void PlatformViewLayoutChange(object? sender, LayoutChangeEventArgs e)
+		{
+			if (PlatformView != null && VirtualView != null)
+			{
+				PlatformView.SetHeight(PlatformView.Height);
```

Comment:
Maybe related with this change, some Entry tests are failing
![image](https://github.com/dotnet/maui/assets/6755973/da535d4e-7cf8-41ae-a5d7-c44ece1f3e66)

Reply:
Hmm strange, they pass for me, and the last one is about verifying screenshots which also looks good to me
<img width="570" alt="Screenshot 2024-06-15 at 01 51 04" src="https://github.com/dotnet/maui/assets/42434498/83de3439-f533-4a45-b60f-a3e255fdb7b3">
![Screenshot_1718409338](https://github.com/dotnet/maui/assets/42434498/17d67dc7-9070-45f9-adab-d1339e09607d)